src/test/java/ru/training/at/hw2/ex2/EpamDemoWebPageTests.java

```diff
@@ -0,0 +1,86 @@
+package ru.training.at.hw2.ex2;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class EpamDemoWebPageTests extends WebPageTestsCommonConditions {
+
+    @Test
+    public void demoPageTests() {
+        //1.Open test site by URL
+        driver.get(HOMEPAGE_URL);
+
+        //2. Assert Browser title
+        String expectedTitle = "Home Page";
+        String actualTitle = driver.getTitle();
+        Assert.assertEquals(actualTitle, expectedTitle);
+
+        //3.Perform login
+        new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS).until(ExpectedConditions
+            .elementToBeClickable(By.className("profile-photo"))).click();
+
+        new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS).until(ExpectedConditions
+            .elementToBeClickable(By.id("name"))).sendKeys(LOGIN_NAME);
+
+        new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS).until(ExpectedConditions
+            .elementToBeClickable(By.id("password"))).sendKeys(PASSWORD);
+
+        new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS).until(ExpectedConditions
+            .elementToBeClickable(By.id("login-button"))).click();
+
+        //4. Assert Username is loggined
+        String actualDisplayedUsername = new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS).until(ExpectedConditions
+            .presenceOfElementLocated(By.id("user-name"))).getText();
+
+        String expectedUsername = "ROMAN IOVLEV";
+        Assert.assertEquals(actualDisplayedUsername, expectedUsername);
+
+        //5.Open through the header menu Service -> Different Elements Page
+        new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS).until(ExpectedConditions
+            .elementToBeClickable(By.xpath("//li[@class=\"dropdown\"]/*[@class=\"dropdown-toggle\"]"))).click();
+
+        new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS).until(ExpectedConditions
+            .elementToBeClickable(By.xpath("//a[contains(@href,\"different-elements.html\")]"))).click();
+
+        //6.Select checkboxes - Water, Wind
+        List<WebElement> checkboxMenuItems = new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS).until(ExpectedConditions
+            .presenceOfAllElementsLocatedBy(By.className("label-checkbox")));
+        for (WebElement el : checkboxMenuItems
+        ) {
```

Comment:
Please use more Javaic breaks :)
```suggestion
        for (WebElement el : checkboxMenuItems) {
```

Reply:
Corrected